.github/workflows/build-dependencies-table.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Dependencies Table
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-table:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install requests pandas
+
+    - name: Fetch attribute table
+      run: |
+        python - <<EOF
+        import requests
+        import pandas as pd
+
+        url = "https://schematic.api.sagebionetworks.org/v1/ui/"  # Endpoint URL
+        jsonld_url = "https://raw.githubusercontent.com/ncihtan/data-models/main/HTAN.model.jsonld"
```

Comment:
If we were to move to being part of the `ci-lint-validate-convert` action we would need to set this URL dynamicaly